package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "react-accessible-dropdown-menu-hook",
-	"version": "2.3.0",
+	"version": "2.4.0",
```

Since I'd consider this a bug fix, we should just increment the patch version number:

```suggestion
	"version": "2.3.1",
```